src/main/java/com/twilio/sdk/resource/InstanceResource.java

```diff
@@ -153,6 +153,16 @@ public Integer getPropertyAsInteger(String name) {
 	}
 
 	/**
+	 * Gets the property as a Boolean value.
+	 *
+	 * @param name property name
+	 * @return Boolean value
+	 */
+	public Boolean getPropertyAsBoolean(String name) {
+		return Boolean.valueOf(getProperty(name));
```

Comment:
That isn't my point. My point is, your implementation of `getPropertyAsBoolean` will NEVER work because `getProperty` will always throw an exception when the data is not a string.

Again:
https://github.com/twilio/twilio-java/blob/ip-messaging-updates/src/main/java/com/twilio/sdk/resource/InstanceResource.java#L104